packages/parsers/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@data-driven-forms/parsers",
+  "version": "0.4.2",
+  "description": "Parsers for Data Driven Forms.",
+  "main": "dist/cjs/index.js",
+  "module": "dist/umd/index.js",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rollup -c ./rollup.config.js",
+    "release": "semantic-release"
```

Comment:
Delete this rule or ignore it in lerna build

Reply:
removed :kimono: